presto-openapi/src/main/resources/queued_statement.yaml

```diff
@@ -46,6 +46,58 @@ paths:
                 $ref: './schemas.yaml/#/components/schemas/QueryResults'
         '400':
           description: Bad request
+    put:
```

Comment:
This should be put under a separate `paths` component which includes the `queryId` in the path. See the example here: https://swagger.io/docs/specification/describing-parameters/

e.g. for this case

```yaml
paths:
  /v1/statement/{queryId}:
    put:
      parameters:
        - name: queryId
          in: path
          .... 
```


While reviewing, I also noticed some other existing paths in this part of the spec are slightly wrong as well, so I will fix them in a separate PR.

Reply:
@ZacBlanco done